src/components/custom/other-platforms.js

```diff
@@ -231,7 +231,7 @@ export const OtherPlatform = ({ header, subHeader, exclude, is_nav, is_ppc_redir
             )}
             <StyledFlexGridContainer content_width="38.4rem" gap="1rem" grid="3" justify="center">
                 {exclude.toLowerCase() !== 'dtrader' && <TraderCard />}
-                <ROW>{exclude.toLowerCase() !== 'dbot' && <BotCard />}</ROW>
+                {exclude.toLowerCase() !== 'dbot' && <BotCard />}
                 {exclude.toLowerCase() !== 'dmt5' && <DMT5Card is_ppc_redirect={is_ppc_redirect} />}
```

Comment:
Not part of your change but can we move `exclude.toLowerCase()` to variable instead?

Reply:
make it `const` instead also it is for readability and ease of development, later you will only need to update it in one place.

Reply:
Ok